src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
+        int randomNumber = new Random().nextInt(101);
```

Comment:
Don't create a new instance of 'Random' every time the method is called. Instead, create a single static instance and reuse it.